src/models/board.ts

```diff
@@ -58,7 +58,7 @@ const getBoardWithComment = async (
 };
 
 const getBoards = async (keyword: string) => {
-  return await prisma.$queryRawUnsafe(`
+  return await prisma.$queryRawUnsafe<{ BoardSearchResult: string }[]>(`
```

Comment:
```suggestion
  return await prisma.$queryRawUnsafe<{ id: string, boardTitle: string, board_contents: string, ~ }[]>(`
```
select 하는 결과 값이 들어가게 해주세요~

Reply:
타입 적는 부분이 커진다면 interface 로 따로 정의하시면 되겠습니다.

Reply:
수정완료했습니다!!